src/client.ts

```diff
@@ -186,9 +188,14 @@ export class FarosClient {
   }
 
   queryParameters(): string | undefined {
```

Comment:
Let's instead of `string | undefined` use object, e.g
```typescript
queryParameters(): Dictionary<any> {
   const params = {};
    if (this.graphVersion === GraphVersion.V2) {
        params['phantoms'] = this.phantoms;
        if (this.visibility) {
            params['visibility'] = this.visibility;
        }
    }
    return params;
}
```
This would propagate upstream. See - https://www.kindacode.com/snippet/axios-passing-query-parameters-in-get-post-requests/